### ISSUE TITLE
Remove wiki article, but substitute with FSM image and brief, consultant-written explanation

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/ember-study#84]
Not sure it's serving as much pedagogical purpose as we would like, considering the amount of time it may take. 015 said it was too long per mention in #82. How do we feel about removing it @jrhorn424 I assume you put it there? Is it worth them spending the time on this before any Ember? Can their time be better spent?

**Comment by MicFin** [comment]
@payne-chris-r this can be closed per https://github.com/ga-wdi-boston/ember-study/pull/89

**Comment by jrhorn424** [comment]
@MicFin No, it cannot. See my comment [here](https://github.com/ga-wdi-boston/ember-study/issues/82#issuecomment-281835130).
